### PR TITLE
Add docs for the UniFi Protect button platform

### PR DESCRIPTION
**Pull request by AngellusMortis**
## Proposed change

Add docs for the UniFi Protect button platform

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/63001
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifiprotect.markdown [hunk 1]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
 
 * **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
 * **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).
+* **Button** - A button entity that is disabled by default will be added for each camera entity. This will let you Reboot your camera device.
+
+### Flood Light
+
+Each UniFi Protect flood light will get the following entities added:
+
+* **Button** - A button entity that is disabled by default will be added for each flood light entity. This will let you Reboot your flood light device.
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Button** - A disabled by default button entity is added for each floodlight entity. The button will let you Reboot your floodlight device.
```

File: source/_integrations/unifiprotect.markdown [hunk 2]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
...
+
+### Sensor
+
+Each UniFi Protect sensor will get the following entities added:
+
+* **Button** - A button entity that is disabled by default will be added for each sensor entity. This will let you Reboot your sensor device.
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Button** - A disabled by default button entity is added for each sensor entity. The button will let you Reboot your sensor device.
```

File: source/_integrations/unifiprotect.markdown [hunk 3]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
...
+
+### Viewport
+
+Each UniFi Protect viewport will get the following entities added:
+
+* **Button** - A button entity that is disabled by default will be added for each viewport entity. This will let you Reboot your viewport device.
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Button** - A disabled by default button entity is added for each viewport entity. The button will let you Reboot your viewport device.
```

**Comment by bdraco** [comment]
I think the only reason to document it is that its disabled by default so they realize they have to go in an enable it if they want it.  I suggest rephrasing to make that the first thing they see.

**Review by bdraco — DISMISSED** [review]

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks @AngellusMortis

File: source/_integrations/unifiprotect.markdown [hunk 4]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
 
 * **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
 * **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).
+* **Button** - A disabled by default button entity is added for each camera entity. The button will let you reboot your camera device.
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown:82]: Do cameras get a button?
Please describe the functionality in the list, not the entity platforms.

File: source/_integrations/unifiprotect.markdown [hunk 5]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
...
+
+### Floodlight
+
+Each UniFi Protect flood light will get the following entities added:
+
+* **Button** - A disabled by default button entity is added for each floodlight entity. The button will let you reboot your floodlight device.
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown:88]: Same as above.

File: source/_integrations/unifiprotect.markdown [hunk 6]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
...
+
+### Sensor
+
+Each UniFi Protect sensor will get the following entities added:
+
+* **Button** - A disabled by default button entity is added for each sensor entity. The button will let you reboot your sensor device.
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown:94]: Same as above. I find this rather confusing actually... is it a button or a sensor?

File: source/_integrations/unifiprotect.markdown [hunk 7]
```diff
@@ -77,6 +79,25 @@ Each UniFi Protect camera will get the following entities added:
...
+
+### Viewport
+
+Each UniFi Protect viewport will get the following entities added:
+
+* **Button** - A disabled by default button entity is added for each viewport entity. The button will let you reboot your viewport device.
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown:100]: Same as above

**Review by frenck — COMMENTED** [review]
Please address these in a new PR.

**Pull request merged**